### PR TITLE
Feature/article create

### PR DESCRIPTION
**Pull request by HumRuslan**
**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

### CODE REVIEW

**Comment by HumRuslan** [comment]
create_error
create_full
create_empty

File: app/controllers/account/admin/article_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    authorize([:account, :admin, @article])
+  end
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
+      authorize([:account, :admin, @article])
+      redirect_to account_admin_article_index_path
+    else
+      skip_authorization
```
Comment (chu4) [inline app/controllers/account/admin/article_controller.rb:18]: why do you skip auth?

File: app/models/article.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Article < ApplicationRecord
+  mount_uploader :picture, PictureUploader
+
+  validates :picture, :headline, :caption, :alt_picture, :content, presence: true
+  enum published: { published: 1, unpublished: 0 }
+
+  before_save :set_default_params
```
Comment (chu4) [inline app/models/article.rb:7]: before_save will run on both create and update

File: db/seeds.rb [hunk 3]
```diff
@@ -1,23 +1,3 @@
...
-if Rails.env.development?
-  User.create(admin)
-  20.times do
-    user = FactoryBot.create :user
-    user.confirm
-  end
-end
-
 
+Dir[File.join(Rails.root, 'db', 'seeds', '*.rb')].sort.each { |seed| load seed }
```
Comment (chu4) [inline db/seeds.rb:3]: very good choice

**Review by chu4 — APPROVED** [review]

**Pull request merged**